### PR TITLE
update readme with future plans

### PR DESCRIPTION
**Pull request by titusfortner**
@twalpole & @kapoorlakshya @luke-hill @p0deje

Does this sound like a reasonable plan for this project? Any suggestions/disagreements?

See merged text here: https://github.com/titusfortner/webdrivers#update-future-of-this-project

### CODE REVIEW

**Comment by titusfortner** [comment]
To avoid "me too," replies, please add a 👍 reaction to this comment if you need a solution to work with Selenium 3. That's the piece that I'm not sure I have time to address in the near term.

**Review by p0deje — APPROVED** [review]

**Comment by twalpole** [comment]
Seems reasonable to me

**Comment by kapoorlakshya** [comment]
Sounds good to me and I like the plan for v6.

File: README.md [hunk 1]
```diff
@@ -5,21 +5,37 @@
...
-* Specified by `driver_path` attribute accessor for the `Service` class (which is what this gem does)
-* Located in a directory included in the `PATH` environment variable
-* Located in the directory Selenium Manager has downloaded it to
+If you can update to to the latest version of Selenium, please do so and stop requiring this gem.
+Selenium 4.10 will work for Chrome v115 (ignore the warning), and Selenium 4.11 will work for everything going forward.
+Please provide feedback or raise issues with [Selenium Project](https://github.com/SeleniumHQ/selenium/issues/new/choose)
 
-If you don't have any drivers in locations on `PATH`, you can try not requiring this gem and see if you still need it.
+If you cannot upgrade to the latest version of Selenium, you can set the required version of chromedriver to v114
+(`Webdrivers.required_version = '114.0.5735.90'`) and
```
Comment (luke-hill) [inline README.md:19]: @titusfortner I think this line of code is wrong - or it was for me.

I used the line `Webdrivers::Chromedriver.required_version = '114.0.5735.90'`

Comment (titusfortner) [reply README.md:19]: oops, you're right.

Comment (titusfortner) [reply README.md:19]: fixed, thanks.

**Pull request merged**